manifest.php

```diff
@@ -47,10 +48,10 @@
     'label' => 'Proctoring',
     'description' => 'Proctoring for deliveries',
     'license' => 'GPL-2.0',
-    'version' => '16.4.0',
+    'version' => '16.4.1',
     'author' => 'Open Assessment Technologies SA',
     'requires' => array(
-        'tao'            => '>=36.0.0',
+        'tao'            => '>=38.3.0',
```

Comment:
if you change a major version of a dependency, then you should up the same version of this extension

Reply:
thx, fixed